mkosi/__init__.py

```diff
@@ -3670,7 +3671,7 @@ def compressor_command(option: Union[str, bool]) -> List[str]:
     elif option is False:
         return ["cat"]
     else:
-        die(f"Unknown compression {option}")
+        die(f"Unknown compression {option}", ArgsException)
```

Comment:
This should stay MkosiException. We only want to use ConfigException for invalid combinations of valid options. This is just wrong values being passed